### PR TITLE
chore: remove bok-choy usage

### PR DESCRIPTION
**Pull request by salman2013**
Description:

As the bok-choy has been deprecated https://github.com/openedx/public-engineering/issues/13, we are removing its usage from the code in this PR.

Ticket : https://github.com/orgs/openedx/projects/55/views/1?pane=issue&itemId=43871848

### CODE REVIEW

**Review by thezaeemaanwar — APPROVED** [review]

**Review by farhan — APPROVED** [review]

File: acceptance_tests/mixins.py
Comment (feanil) [inline acceptance_tests/mixins.py:1]: It looks like a lot of the mixins in this file are also not in use anywhere else.  Can you search for them in the rest ofthe repo and remove any mixins that are not of any use.

**Review by feanil — COMMENTED** [review]
There's some more mixins in the acceptance test module that should also be cleaned up because they are no longer being used anywhere but other than that this looks good.

File: acceptance_tests/mixins.py
Comment (salman2013) [reply acceptance_tests/mixins.py:1]: @feanil I have further clean up the mixin in acceptance test module, could you please take a look into it, thanks

File: requirements/test.in [hunk 1]
```diff
@@ -4,7 +4,6 @@
 -r base.txt
 
 astroid
-bok-choy
```
Comment (feanil) [inline requirements/test.in:7]: Do we still need selenium in this repo or should that be removed as well?

**Review by feanil — COMMENTED** [review]
Sorry I missed one more thing but then I think this is good to go.

File: requirements/test.in (hunk 1 above)
Comment (salman2013) [reply requirements/test.in:7]: It does not, i searched and removed it. thanks for pointing this.

**Review by feanil — APPROVED** [review]

**Pull request merged**